File: cakk-api/src/main/java/com/cakk/api/dto/response/shop/CakeShopDetailResponse.java
package com.cakk.api.dto.response.shop;

import java.util.Set;

import lombok.Builder;

import com.cakk.common.enums.Days;
import com.cakk.domain.dto.param.shop.CakeShopLinkParam;

@Builder
public record CakeShopDetailResponse(
	Long cakeShopId,
	String cakeShopName,
	String thumbnailUrl,
	String cakeShopBio,
	String cakeShopDescription,
	Set<Days> operationDays,
	Set<CakeShopLinkParam> links
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/shop/CakeShopDetailResponse.kt
package com.cakk.core.dto.response.shop

import com.cakk.common.enums.Days
import com.cakk.infrastructure.persistence.param.shop.CakeShopLinkParam

data class CakeShopDetailResponse(
    val cakeShopId: Long,
    val cakeShopName: String,
    val thumbnailUrl: String,
    val cakeShopBio: String,
    val cakeShopDescription: String,
    val operationDays: Set<Days>,
    val links: Set<com.cakk.infrastructure.persistence.param.shop.CakeShopLinkParam>
)
